torchao/quantization/quantize_/workflows/int4/int4_tile_packed_to_4d_tensor.py

```diff
@@ -76,6 +84,7 @@ def from_hp(
         cls,
         hp_tensor: torch.Tensor,
         block_size: List[int],
+        use_hqq: bool = False,
```

Comment:
have you thought of making it non binary but enum based? 

default
hqq

so that we can expand if necessary

Reply:
we haven't seen new updates to this one in that past year, I feel it's unlikely we will get more algorithms here. we can also make hqq default and remove this flag as well I feel